File: Backend/DTOs/Challenge/ChallengeQuestionDTO.cs
﻿namespace Learnz.DTOs;
public class ChallengeQuestionDTO
{
    public string Question { get; set; }
    public string? Description { get; set; }
    public QuestionType QuestionType { get; set; }
    public List<ChallengeQuestionAnswerDTO>? AnswerSetOne { get; set; }
    public List<ChallengeQuestionAnswerDTO>? AnswerSetTwo { get; set; }
}

File: Frontend/src/app/DTOs/Challenge/ChallengeQuestionDTO.ts
import { QuestionType } from "src/app/Enums/QuestionType";
import { ChallengeQuestionAnswerDTO } from "./ChallengeQuestionAnswerDTO";

export interface ChallengeQuestionDTO {
    question: string;
    description: string | null;
    questionType: QuestionType;
    answerSetOne: ChallengeQuestionAnswerDTO[] | null;
    answerSetTwo: ChallengeQuestionAnswerDTO[] | null;
}